src/sqs/queues.ts

```diff
@@ -68,20 +70,46 @@ const sqsQueues = {
 	),
 
 	start: () => {
+		logger.info("Starting queues");
 		sqsQueues.backfill.start();
 		sqsQueues.push.start();
 		sqsQueues.discovery.start();
 		sqsQueues.deployment.start();
 		sqsQueues.branch.start();
+		logger.info("All queues started");
 	},
 
-	stop: () => {
-		sqsQueues.backfill.stop();
-		sqsQueues.push.stop();
-		sqsQueues.discovery.stop();
-		sqsQueues.deployment.stop();
-		sqsQueues.branch.stop();
-	}
-}
+	stop: async () => {
+		logger.info("Stopping queues");
+		await Promise.all([
+			sqsQueues.backfill.stop(),
+			sqsQueues.push.stop(),
+			sqsQueues.discovery.stop(),
+			sqsQueues.deployment.stop(),
+			sqsQueues.branch.stop()
+		]);
+		logger.info("All queues stopped");
+	},
 
-export default sqsQueues
+	purge: async () => {
+		logger.info("Purging queues");
+		await Promise.all([
+			sqsQueues.backfill.purgeQueue(),
+			sqsQueues.push.purgeQueue(),
+			sqsQueues.discovery.purgeQueue(),
+			sqsQueues.deployment.purgeQueue(),
+			sqsQueues.branch.purgeQueue()
+		]);
+		logger.info("All queues purged");
+	},
+
+	getMessageCount: async () => {
```

Comment:
Can we add comments that it is test-only?

Reply:
I removed them